src/adapters/tapsense.js

```diff
@@ -0,0 +1,89 @@
+//v0.0.1
+var bidfactory = require('../bidfactory.js');
+var bidmanager = require('../bidmanager.js');
+var adloader = require('../adloader');
+var utils = require('../utils.js');
+
+var TapSenseAdapter = function TapSenseAdapter() {
+  var version = "0.0.1";
+  var creativeSizes = [
+    "320x50"
+  ];
+  var validParams = [
+    "ufid",
+    "refer",
+    "ad_unit_id",
+    "device_id",
+    "lat",
+    "long",
+    "user",
+    "price_floor",
+    "test",
+    "jsonp"
+  ];
+  var bids;
+  window.tapsense = {};
+  function _callBids(params) {
+    bids = params.bids || [];
+    for (var i = 0; i < bids.length; i++) {
+      var bid = bids[i];
+      var isValidSize = false;
+      if (!bid.sizes) {
+        return;
+      }
+      for (var k = 0; k < bid.sizes.length; k++) {
+        if (creativeSizes.indexOf(bid.sizes[k].join("x")) > -1) {
```

Comment:
This fails if the `bid.sizes` is a single array and not a nested array (ie `[300,250];` instead of `[[300,250]];`)